go/client/cmd_login.go

```diff
@@ -141,6 +172,17 @@ func (c *CmdLogin) ParseArgv(ctx *cli.Context) error {
 			c.clientType = keybase1.ClientType_GUI
 		}
 	}
+
+	if ctx.Bool("stdin") {
+		stdinBytes, err := ioutil.ReadAll(os.Stdin)
+		if err != nil {
+			return err
+		}
+		c.stdinPassphrase = strings.TrimSpace(string(stdinBytes))
```

Comment:
This will break people whose passwords begin or end with spaces. Weirdly enough, such people exist!  It cost us a bunch of BTC to figure it out.  Long story.